src/entries-list/node-card.ts

```diff
@@ -0,0 +1,266 @@
+import { LitElement, html, css } from "lit";
+import { customElement, property, query } from "lit/decorators.js";
+import { classMap } from "lit/directives/class-map.js";
+import { ConfiguredEntry } from "../api/list-entries";
+import { ActionDetail } from "@material/mwc-list/mwc-list-foundation";
+import "@material/mwc-button";
+import "@material/mwc-icon-button";
+import "@material/mwc-dialog";
+import { Dialog } from "@material/mwc-dialog";
+import "../components/esphome-button-menu";
+import "../components/esphome-card";
+import "@polymer/paper-tooltip/paper-tooltip.js";
+import { openCleanMQTTDialog } from "../clean-mqtt";
+import { openCleanDialog } from "../clean";
+import { deleteConfiguration } from "../api/configuration";
+import { fireEvent } from "../util/fire-event";
+import { openValidateDialog } from "../validate";
+import { openEditDialog } from "../legacy";
+import { openInstallDialog } from "../install-update";
+import { openLogsTargetDialog } from "../logs-target";
+
+@customElement("esphome-node-card")
+class ESPHomeNodeCard extends LitElement {
+  @property() public entry!: ConfiguredEntry;
+  @property() public onlineStatus!: boolean;
+  @property() public highlighted!: boolean;
```

Comment:
Use present, not past tense

```suggestion
  @property() public highlight = false;
```